main.js

```diff
@@ -0,0 +1,6 @@
+// wrong function is odd checker

```

Comment:
remove this single comment after fixing the function below correctly

Reply:
Resolved